docs/manual/extensions/cca-merger2.en.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Merger²"
+description: "Merger² is a frontend module to display modules/contents under certain conditions."
+url: "extension/merger2"
+---
+
+**[contao-community-alliance/merger2](https://packagist.org/packages/contao-community-alliance/merger2)**
+
+_von [Contao Community Alliance (CCA)](https://c-c-a.org/)_
+
+
+
+The Merger² extension for the CMS Contao offers a powerful frontend (FE) module to add content under certain conditions.
+Display or merge conditions. The contents can be articles, pages or other FE modules. The Conditions for the display
```

Comment:
Hier fehlt wohl etwas. Evt. eher: "... under certain conditions, like display or merge conditions".

"The Conditions" -> "The conditions"